### DISCUSSION TITLE
prompt flow and Azure WebApp application

### DISCUSSION THREAD

**Discussion by rrmehdi** [community/community#67506] [Programming Help]
HI, I have a question of more general character, from conceptual point of view.  PromptFlow looks great and looks like a nice way in automating the prompt engineering process. How then this prompt flow is impacting Azure WebApp application that's targeting a more general audience. in other words, wow to orchestrate a transition from prompt flow product to  WebApp deployment that consumes LLM, ACS and show the responses. Is it well understood from developers' side?

**Comment by sethjuarez** [comment]
Great question! Prompt flows are deployed as endpoints: think of them as wrapped OpenAI calls with special functionality defined in a graph (DAG to be specific). So, in the case of Web Apps, these become another resource that can be called / used.

**Comment by JonRippon** [comment]
Also looking for guidance on how we consume the deployment endpoint, say in a web app or ideally via Teams App? Is a Function App  required in the middle?  

Thanks
